cache/caffeine-cache/src/it/test-caffeine-cache/pom.xml

```diff
@@ -147,6 +147,7 @@
                                 <exclude>**/ConstructorMapperPgTest.*</exclude>
                                 <exclude>**/TestArgumentBinder.*</exclude>
                                 <exclude>**/TestQueriesPG.*</exclude>
```

Comment:
Is it possible to hold this configuration somehow by annotating the code? It sucks to edit three pom files just to add a new test case.

Reply:
will all be gone once JDK8 is gone